newdle/templates/deletion_email.txt

```diff
@@ -0,0 +1,7 @@
+Hi,
+
+{{ creator }} has deleted the newdle "{{ title }}".
+
+{% if comment %}
+    Comment: {{ comment }}
+{% endif %}
```

Comment:
Otherwise you get indentation and extra blank lines in the resulting email:

```suggestion
{%- if comment %}

Comment: {{ comment }}
{%- endif -%}
```